FirebaseFirestoreUI/FUISnapshotArrayDiff.m

```diff
@@ -246,7 +246,7 @@ - (instancetype)initWithInitialArray:(NSArray *)initialArray resultArray:(NSArra
   if (self != nil) {
     _initial = [initialArray copy];
     _result = [resultArray copy];
-    [self buildDiffs];
+    [self buildFastDiffs];
```

Comment:
Is buildDiffs no longer used?  Should it be removed or should there be a comment explaining if/when we might want to revive it?

Reply:
It's dead code and will be removed.